alembic/versions/1d0bb7fede17_add_pipeline_run_status.py

```
@@ -0,0 +1,33 @@
+
+Revision ID: 1d0bb7fede17
+Revises: 482cd6517ce4
+Create Date: 2025-05-19 10:58:15.993314
+"""
+
+from typing import Sequence, Union
+
+from alembic import op
+
+from cognee.infrastructure.databases.relational.get_relational_engine import get_relational_engine
+from cognee.modules.pipelines.models.PipelineRun import PipelineRun, PipelineRunStatus
```

Ruff (0.11.9)

`cognee.modules.pipelines.models.PipelineRun.PipelineRun` imported but unused

Remove unused import

(F401)